cronet/build.gradle

```diff
@@ -33,7 +33,9 @@ android {
 }
 
 dependencies {
-    implementation project(':grpc-core')
+    api project(':grpc-core'),
+            libraries.cronet_api
```

Comment:
There is another `cronet_api` down below for `implementation`. It is controversial whether it should be `api` or `implementation` (IMO it doesn't matter much). I am ok with either. Similar situation applies for some other modules such as `grpc-okhttp` (for `okhttp`) and `grpc-protobuf` (for `grpc-protobuf-lite`).

Reply:
Update: for `grpc-okhttp` probably not reason to expose `okhttp` as users do not interact with okhttp APIs directly.

Reply:
Oh, the `implementation libraries.cronet_api` should be removed because it's redundant now.

Reply:
For grpc-okhttp, it needs okhttp as api dep because it is part of public API `OkHttpChannelBuilder.connectionSpec(com.squareup.okhttp.ConnectionSpec connectionSpec)` 

>Update: for grpc-okhttp probably not reason to expose okhttp as users do not interact with okhttp APIs directly.

Reply:
Oh I see. Missed that.